ts/screens/wallet/AddManagerToCardScreen.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { FlatList, Image, StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import AppHeader from "../../components/ui/AppHeader";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import { transactionManager } from "../../types/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const paymentManagers: ReadonlyArray<
+  transactionManager
+> = WalletAPI.getManagers();
+
+const style = StyleSheet.create({
+  listItem: {
+    marginLeft: 0,
+    flex: 1,
+    paddingRight: 0
+  },
+
+  icon: {
+    flexDirection: "row",
+    alignItems: "center"
+  },
+
+  feetext: {
+    color: variables.brandDarkGray,
+    fontSize: variables.fontSize2
+  }
+});
+
+/**
+ * Selection of a payment manager for the new added card
+ */
+export class AddManagerToCardScreen extends React.Component<Props, never> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  public render(): React.ReactNode {
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("saveCard.saveCard")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content>
+          <H1>{I18n.t("wallet.AddManager.title")}</H1>
+          <View spacer={true} />
+          <Text>
+            {I18n.t("wallet.AddManager.info") + " "}
+            <Text bold={true}>
+              {I18n.t("wallet.AddManager.infobold") + " "}
+            </Text>
+            <Text>{I18n.t("wallet.AddManager.info2") + " "}</Text>
+            <Text link={true}>{I18n.t("wallet.AddManager.link")}</Text>
+          </Text>
+          <View spacer={true} />
+          <FlatList
+            ItemSeparatorComponent={() => (
+              <View
+                style={{
```

Comment:
no inline styles
@cloudify have you considered adding a lint rule like [no-inline-styles](https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/no-inline-styles.md)? This would probably be of help for new contributors :)